### PR TITLE
Milestone 2# Add Components

### PR DESCRIPTION
**Pull request by gscarv13**
### This pull request contains the following
- Created a `components` dedicated folder inside src 
- Added Button, Display, and ButtonPanel components
- Define PropTypes to Button and Display components

### CODE REVIEW

File: src/components/App.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+import Display from './Display';
+import ButtonPanel from './ButtonPanel';
+
+function App() {
+  return (
+    <div>
+      <p>Calculator</p>
+      <Display />
+      <ButtonPanel />
+    </div>
```
Comment (bellom) [inline src/components/App.js]: You need to wrap both `Display` and `ButtonPanel` with [React.Fragment](https://reactjs.org/docs/fragments.html#short-syntax)
Reason:
We don't want to create an extra element, that's why you need to use fragment, so we are able to render without having to create an extra node wrapping `Display` and `ButtonPanel`.

File: src/components/ButtonPanel.js [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+        <Button name="2" />
+        <Button name="3" />
+        <Button name="+" />
+      </div>
+      <div>
+        <Button name="0" />
+        <Button name="." />
+        <Button name="=" />
+      </div>
+    </>
```
Comment (bellom) [inline src/components/ButtonPanel.js]: ### [Optional]
Your code appears to be WET-(Write Every Time), Microverse encourage DRY-(Don't Repeat Yourself) code, In your case; you repeated the `Button` component multiple times.

You can use [objects.key](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/keys)  and `map` to loop through the objects. `objects.keys` for the outer loop and `map` for the inner loop.

Example; 
```
const GroupButtons = {
  Group 1: ['AC', '+/-', '%', '÷'],
  Group 2: ['7', '8', '9', 'X'],
   ............
}
```

**Review by bellom — CHANGES_REQUESTED** [review]
## Hi @gscarv13  :clap: 

### Awesome job so far!! :+1: 

#### You have done wonderfully well, you passed `almost` all the checklist for this project :+1: just a few issues left to work on............ :slightly_smiling_face: 

# Status: Changes Required! ♻️ 

**Kindly check out my comments below;**

### ♻️ Kindly re-submit for review after making all those changes :slightly_smiling_face: Thumps -up await you.....

**Best**

File: src/components/ButtonPanel.js (hunk 2 above)
Comment (gscarv13) [reply src/components/ButtonPanel.js]: ### **Thanks a lot!**

I couldn't think of a way to optimize it, really appreciated :+1: :+1:

**Review by enaburekhan — APPROVED** [review]
# Status: Approve:heavy_check_mark:
**Hi Silva!
Congratulations:clap::clap:. Thank you for implementing the changes requested by the last reviewer.** **Move on with the next milestone:walking:**.

Do have questions/comments? kindly drop them :point_down: and ping me on slack @ Eric Enaburekhan to respond.
Happy hacking:blush:
Eric Enaburekhan.

**Pull request merged**